app/controllers/time_entries_controller.rb

```diff
@@ -79,7 +79,7 @@ def stop_time
 
   def update
     if @time_entry.update(time_entry_params)
-      redirect_to time_entries_path
+      redirect_to_entries_on(@time_entry.start_time.to_date)
```

Comment:
For this, I was thinking of something a bit more simplistic, like:
```ruby
redirect_to time_entries_path(date: @time_entry.start_time.to_date)
```

The downside to this, of course, is that it doesn't hide the date, like you mentioned. That said, I'll take simplicity of implementation over that :smiley: